plugins/blip/basics/components/misc/SummaryGroup.js

```diff
@@ -59,23 +61,24 @@ var SummaryGroup = React.createClass({
       </div>
     );
   },
-  renderInfo: function(option) {
+  renderInfo: function(option, id, options) {
     var self = this;
     var classes = classnames({
       'SummaryGroup-info--selected': (option.key === self.props.selectedSubtotal),
       'SummaryGroup-info-primary': option.primary,
-      'SummaryGroup-info': !option.primary
+      'SummaryGroup-info': !option.primary,
+      'SummaryGroup-info-tall': ( !option.primary && options.length <= 3 )
     });
 
     var value = (
       <span className="SummaryGroup-option-count">
         {option.key === 'total'? this.props.data[option.key] :
-          this.props.data[option.key].count}
+          this.props.data[option.key].count || 0}
       </span>
     );
-    var percentage = option.key === 'total' ? null : (
+    var percentage = (!option.percentage) ? null : (
       <span className="SummaryGroup-option-percentage">
-        {d3.format('%')(this.props.data[option.key].percentage)}
+        {d3.format('%')(this.props.data[option.key].percentage || 0)}
```

Comment:
Were you just annoyed with the NaNs you were seeing? Not sure if we should replace missing percentages with 0...